### PR TITLE
Kotlin cleanup, improving nullability

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
For Scoped-Storage, I'm trying to improve notification to be able to use it outside of the app. In the mean-time, I want to add some non-nullability request. And this nullable message was in my path.

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
I'm okay with this but require a stronger Kotlin person to give it a look :pray:

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt [hunk 1]
```diff
@@ -274,19 +274,18 @@ open class DeckPicker :
...
+    private class ImportAddListener(deckPicker: DeckPicker) : TaskListenerWithContext<DeckPicker, String, ImporterData?>(deckPicker) {
+        override fun actualOnPostExecute(context: DeckPicker, result: ImporterData?) {
             if (context.mProgressDialog != null && context.mProgressDialog!!.isShowing) {
                 context.mProgressDialog!!.dismiss()
             }
-            // If result.second and result are both set, we are signalling
+            // If result.errFlag and result are both set, we are signalling
             // some files were imported successfully & some errors occurred.
-            // If result.first is null & result.second & result.third is set
+            // If result.first is null & result.errFlag & result.errList is set
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt:284]:
```suggestion
            // If result.impList is null & result.errList is set
```

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt]: I am rebase-editing this and I incorporated your change

**Review by mikehardy — APPROVED** [review]
LGTM still, I resolved Brayan's comments, ready to go

**Pull request merged**